strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/AuthenticatorsConfigController.java

```diff
@@ -28,63 +25,72 @@
         extends BaseController
 {
 
-    @Inject
-    private AuthenticatorsRegistry authenticatorsRegistry;
+    static final String SUCCESSFUL_REORDER = "Reorder succeeded.";
+    static final String FAILED_REORDER = "Could not reorder authenticators registry.";
+
+    static final String SUCCESSFUL_RELOAD = "Reload succeeded.";
+    static final String FAILED_RELOAD = "Could not reload authenticators registry.";
 
-    @Inject
-    private AuthenticatorsScanner authenticatorsScanner;
+    private final AuthenticatorsRegistry authenticatorsRegistry;
+
+    private final AuthenticatorsScanner authenticatorsScanner;
+
+    public AuthenticatorsConfigController(AuthenticatorsRegistry authenticatorsRegistry,
+                                          AuthenticatorsScanner authenticatorsScanner)
+    {
+        this.authenticatorsRegistry = authenticatorsRegistry;
+        this.authenticatorsScanner = authenticatorsScanner;
+    }
 
     @ApiOperation(value = "Enumerates ordered collection of authenticators with order number and name")
-    @ApiResponses(value = { @ApiResponse(code = 200,
-            message = "The list was returned successfully.") })
-    @RequestMapping(value = "/", method = RequestMethod.GET, produces = { MediaType.APPLICATION_XML_VALUE,
-                                                                          MediaType.APPLICATION_JSON_VALUE })
-    public ResponseEntity<AuthenticatorsRegistry> list()
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "The list was returned successfully.") })
+    @GetMapping(value = "/",
+            consumes = MediaType.APPLICATION_JSON_VALUE,
```

Comment:
Cosmetic: indention should be aligned here as well as in the other methods below :)